taxonomies/ingredients.txt

```diff
@@ -11540,7 +11540,7 @@ bg:Бета каротин
 ca:beta-carotè, beta-carotens,betacarotè
 cs:beta-karoten
 da:Betakaroten
-de:Beta-Carotin
+de:Beta-Carotin, Vitamin A, Vit. A
```

Comment:
this should probably go in the vitamin A entry just above beta carotene.

Reply:
Okay, seems like I overlooked the Vitamin entry. I was wondering why there "were" none.

Reply:
`Vit.` also can't be handled with the taxonomies because of the `.`. It needs to be expanded in code.

Reply:
Okay, so I should delete all `Vit. X` entries, right? We should probably explain that to people adding Ingredients because I saw it this way on the packeting.

Reply:
hmm, it looks like there's no german section of %abbreviations in lib/ProductOpener/Ingredients.pm yet.

Reply:
Right, we will handle "Vit. X" entries during parsing. We have some code in place to expand abbrevations: https://github.com/openfoodfacts/openfoodfacts-server/blob/master/lib/ProductOpener/Ingredients.pm#L261